src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/courtsentencing/CourtSentencingMigrationIntTest.kt

```diff
@@ -676,6 +683,66 @@ fun dpsMigrationCreateResponseWithTwoAppearancesAndTwoCharges(): MigrationCreate
     sentenceTerms = sentenceTermIds,
   )
 }
+fun dpsBookingCloneCreateResponseWithTwoAppearancesAndTwoCharges(): BookingCreateCourtCasesResponse {
```

Comment:
Same concept for this test data - identical to existing migration response other than the types